src/d/actor/d_a_majuu_flag.cpp

```diff
@@ -6,55 +6,794 @@
 #include "d/actor/d_a_majuu_flag.h"
 #include "d/d_procname.h"
 #include "d/d_priority.h"
+#include "d/d_kankyo_wether.h"
+#include "d/d_com_inf_game.h"
+#include "f_op/f_op_actor_mng.h"
+#include "d/d_s_play.h"
+#include "d/res/res_matif.h"
+#include "d/res/res_vsvfg.h"
+#include "d/res/res_xhcf.h"
+#include "d/res/res_cloth.h"
+#include "dolphin/gx/GXTexture.h"
+#include "m_Do/m_Do_graphic.h"
+#include "m_Do/m_Do_hostIO.h"
+
+class daMajuu_Flag_HIO_c : public mDoHIO_entry_c {
+public:
+    daMajuu_Flag_HIO_c() {
+        mbUseToonTex = true;
+        m06 = 0;
+        m05 = 0;
+        mWaveSpeed = 0x400;
+        mWindSpeed1 = 20.0f;
+        mWindSpeed2 = 10.0f;
+        mFlagScale = 1.0f;
+        mbHasCustomRotation = false;
+        mRotX = -0x5DC;
+        mTransZ = 50.0f;
+        mGravity = -1.25f;
+        mSpring = 0.45f;
+        m07 = 0;
+        m08 = 0;
+    }
+    virtual ~daMajuu_Flag_HIO_c() {}
+
+    void genMessage(JORMContext* ctx);
+
+public:
+    /* 0x04 */ bool mbUseToonTex;
+    /* 0x05 */ s8 m05;
+    /* 0x06 */ s8 m06;
+    /* 0x07 */ s8 m07;
+    /* 0x08 */ s8 m08;
+    /* 0x0C */ s32 mWaveSpeed;
+    /* 0x10 */ f32 mWindSpeed1;
+    /* 0x14 */ f32 mWindSpeed2;
+    /* 0x18 */ f32 mGravity;
+    /* 0x1C */ f32 mSpring;
+    /* 0x20 */ f32 mFlagScale;
+    /* 0x24 */ bool mbHasCustomRotation;
+    /* 0x28 */ cXyz mCustomRotation;
+    /* 0x34 */ s32 mRotX;
+    /* 0x38 */ f32 mTransZ;
+};
+
+static daMajuu_Flag_HIO_c l_HIO;
+
+#include "weak_data_1811.h" // IWYU pragma: keep
+
+static u8 unused1[16] = { 0 };
+
+#include "assets/l_flag02TEX.h"
+
+static Vec l_majuu_flag_pos[] = {
+    { 0.0f, -255.0f, 1300.0f },
+    {
+    0.0f,
+    -306.0f,
+    1040.0f
+    },
```

Comment:
Maybe run "Format Selection" on these long arrays (l_majuu_flag_pos l_texCoord rel_pos_idx_tbl rel_pos_idx_tbl) to fix the weird whitespace

Reply:
That's because this repo uses the vscode-clangd extension instead of C++ tools.